### PR TITLE
Clean ssl qemu conf for without ssl case and refine code

### PR DESCRIPTION
**Pull request by czhang0**
For spice with ssl case, spice tls should be configured in
qemu conf file, while for spice without ssl case, no spice
tls configuration is expected.

For environment restore, qemu and libvirtd should be restored
first, then restore vm.

So refine code to achieve above logicality.

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_domdisplay.py [hunk 1]
```diff
@@ -48,7 +48,11 @@ def run(test, params, env):
              "spice" in options and graphic != "spice":
             status_error = True
 
-    def prepare_ssl_env():
+    def restart_libvirtd():
+        # make modification effect
+        utils_libvirtd.libvirtd_restart()
```
Comment (nasastry) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domdisplay.py]: usage of 'libvirtd_restart()' is deprecated

```
def libvirtd_restart():
    libvirtd_instance = Libvirtd()
    deprecation_warning()
    return libvirtd_instance.restart()
```

```
def deprecation_warning():
    """
    As the utils_libvirtd.libvirtd_xxx interfaces are deprecated,
    this function are printing the warning to user.
    """
    logging.warning("This function was deprecated, Please use "
                    "class utils_libvirtd.Libvirtd to manage "
                    "libvirtd service.")
```

You can try some thing like this
```
            libvirtd = utils_libvirtd.Libvirtd()
            libvirtd.restart()
```

Comment (czhang0) [reply libvirt/tests/src/virsh_cmd/domain/virsh_domdisplay.py]: I notice that, thanks for your reminder! I will modify it.

**Comment by czhang0** [comment]
Two cases snipped test result of CI are given below:
=================================================================
job  L0321 INFO | Command line: /root/.libvirt-ci-venv-ci-runtest-NZsBbm/bin/avocado run --vt-type libvirt --vt-machine-type i440fx virsh.domdisplay.have_passwd.no_options.positive_readonly.spice_t.with_ssl --vt-connect-uri qemu:///system
test L0750 INFO | PASS 1-type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.no_options.positive_readonly.spice_t.with_ssl
=================================================================
job  L0321 INFO | Command line: /root/.libvirt-ci-venv-ci-runtest-NZsBbm/bin/avocado run --vt-type libvirt --vt-machine-type i440fx virsh.domdisplay.have_passwd.no_options.positive_readonly.spice_t.without_ssl.domain_id --vt-connect-uri qemu:///system
L0750 INFO | PASS 1-type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.no_options.positive_readonly.spice_t.without_ssl.domain_id

**Comment by czhang0** [comment]
Two cases snipped test result of CI are given below:
job  L0321 INFO | Command line: /root/.libvirt-ci-venv-ci-runtest-NZsBbm/bin/avocado run --vt-type libvirt --vt-machine-type i440fx virsh.domdisplay.have_passwd.no_options.positive_readonly.spice_t.with_ssl --vt-connect-uri qemu:///system
test L0750 INFO | PASS 1-type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.no_options.positive_readonly.spice_t.with_ssl
job  L0321 INFO | Command line: /root/.libvirt-ci-venv-ci-runtest-NZsBbm/bin/avocado run --vt-type libvirt --vt-machine-type i440fx virsh.domdisplay.have_passwd.no_options.positive_readonly.spice_t.without_ssl.domain_id --vt-connect-uri qemu:///system
L0750 INFO | PASS 1-type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.no_options.positive_readonly.spice_t.without_ssl.domain_id

**Pull request merged**